.github/workflows/pull-requests.yml

```diff
@@ -71,8 +88,25 @@ jobs:
         uses: actions/setup-node@v3
         with:
           node-version: '16.14.1'
-        
-      - name: resolve dependency
+
+      - name: Resolve dependency
+        id: cache-npm
+        uses: actions/cache@v3
+        env:
+          cache-name: cache-node-modules
+        with:
+          path: |
+            ~/.npm
+            node_modules
+          key: ${{ runner.os }}-generate-docs-${{ env.cache-name }}-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: |
+            ${{ runner.os }}-generate-docs-${{ env.cache-name }}-
+            ${{ runner.os }}-generate-docs-
+            ${{ runner.os }}-
+
+      - if: ${{ steps.cache-npm.outputs.cache-hit != 'true' }}
+        name: List the state of node modules
+        continue-on-error: true
```

Comment:
This is a different cache. The earlier one was `node_modules` of the app; this is for the global installation of `graphql-inspector`. I don't think so we need caching here. Please revert this.